atorch/README.md

```diff
@@ -1,25 +1,114 @@
+
+   <h3> <a href="https://www.placeholder.com/"> Paper </a> |
+   <a href="https://www.placeholder.com/"> Documentation </a> |
+   <a href="https://www.placeholder.com/"> Examples </a> |
+   <a href="https://www.placeholder.com/"> Blog </a></h3>
+
+   [![GitHub Repo stars](https://img.shields.io/github/stars/intelligent-machine-learning/dlrover?style=social)](https://github.com/intelligent-machine-learning/dlrover/stargazers)
+   [![Build](https://github.com/intelligent-machine-learning/dlrover/actions/workflows/main.yml/badge.svg)](https://github.com/intelligent-machine-learning/dlrover/actions/workflows/main.yml)
+   [![CodeFactor](https://www.codefactor.io/repository/github/intelligent-machine-learning/dlrover/badge)](https://www.codefactor.io/repository/github/intelligent-machine-learning/dlrover)
+   [![HuggingFace badge](https://img.shields.io/badge/%F0%9F%A4%97HuggingFace-Join-yellow)](https://huggingface.co/)
+   [![slack badge](https://img.shields.io/badge/Slack-join-blueviolet?logo=slack&amp)](https://join.slack.com/)
+   [![WeChat badge](https://img.shields.io/badge/微信-加入-green?logo=wechat&amp)](https://huggingface.com/)
+
+
+   | [English](README.md) | [中文](docs/README-zh-Hans.md) |
+
+</div>
+
+## Latest News
+* TODO
+
+## Table of Contents
+<ul>
+ <li><a href="#Why-ATorch">Why ATorch</a> </li>
+ <li><a href="#Features">Features</a> </li>
+ <li>
+   <a href="#ATorch-Applications">ATorch Applications</a>
+   <ul>
+     <li><a href="#ATorchPretrain">ATorch: Super backend engine supportted Ant LLMs training with over 60% HFU</a></li>
+   </ul>
+ </li>
+ <li>
+   <a href="#Parallel-Training-Demo">Parallel Training Demo</a>
+   <ul>
+     <li><a href="#LLaMA2">LLaMA2</a></li>
+     <li><a href="#GPT-3">GPT-3</a></li>
+   </ul>
+ </li>
+ <li>
+   <a href="#Single-GPU-Training-Demo">Single GPU Training Demo</a>
+   <ul>
+     <li><a href="#LLaMA2-Single">GPT-2</a></li>
+   </ul>
+ </li>
+ <li>
+   <a href="#Installation">Installation</a>
+   <ul>
+     <li><a href="#PyPI">PyPI</a></li>
+     <li><a href="#Install-From-Source">Install From Source</a></li>
+   </ul>
+ </li>
+ <li><a href="#Community">Community</a></li>
+ <li><a href="#Contributing">Contributing</a></li>
+ <li><a href="#Cite-Us">Cite Us</a></li>
+</ul>
+
+## Why ATorch
+ATorch is an extension library of PyTorch developed by Ant Group's AI Infrastructure team. It provides efficient and easy-to-use capabilities for model training and optimization. The design principle is to minimally disrupt the native PyTorch programming style. Through its API interfaces, ATorch provides performance optimizations in aspects such as I/O, preprocessing, computation, and communication (including automatic optimization). ATorch has supported large-scale pretraining of LLMs with over 100 billion parameters and 1,000 A100/H100 GPUs. We aim to open source it and make these capabilities reproducible for everyone. We also welcome contributions.
```

Comment:
`Through its API interfaces` -> `Through its API` because API stands for Application Programming Interface.
`1,000 A100/H100 GPUs` -> `thousands of A100/H100 GPUs`.